low-level/framework/utils/iem.py

```diff
@@ -174,3 +174,6 @@ def generate_iem(module, event_code, severity, description):
         except EventMessageError as e:
             logger.error("Failed to send IEM alert."
                          f"Error:{e}")
+        except Exception as err:
```

Comment:
You can put them together since we are not doing anything different in both cases. 
```
except (EventMessageError, Exception) as e:
    logger.error("Failed to send IEM alert."f"Error:{e}")
```

Reply:
Done.